Description: Don't use gendh
 Don't use gendh as it doesn't exist anymore. Replaced by dhparam
Author: Sophie Brun <sophie@freexian.com>
Bug-Kali: https://bugs.kali.org/view.php?id=4137
Last-Update: 2017-08-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/hostapd/certs/Makefile
+++ b/hostapd/certs/Makefile
@@ -49,7 +49,7 @@ passwords.mk: server.cnf ca.cnf client.c
 #
 ######################################################################
 dh:
-	openssl gendh -out dh -2 $(DH_KEY_SIZE)
+	openssl dhparam -out dh $(DH_KEY_SIZE)
 
 ######################################################################
 #
